packages/fxa-settings/src/pages/SignoutSync/index.tsx

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import AppLayout from '../../components/AppLayout';
+import { RouteComponentProps } from '@reach/router';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { usePageViewEvent } from '../../lib/metrics';
+import CardHeader from '../../components/CardHeader';
+import { REACT_ENTRYPOINT } from '../../constants';
+
+export const viewName = 'legal';
+
+const SignoutSync = (_: RouteComponentProps) => {
+  usePageViewEvent(viewName, REACT_ENTRYPOINT);
+
+  // TODO: Add button to make this more automatic. We need the android signout fix
+  //       to be rolled out first though.
+
+  return (
+    <AppLayout>
+      <CardHeader
+        headingTextFtlId="signout-sync-header"
+        headingText="Session Expired"
+      />
+      <section className="flex flex-cols items-center">
+        <FtlMsg id="signout-sync-session-expired">
+          <p className="text-sm">
+            Your account session for sync has expired. Please sign out of sync
```

Comment:
Hm, we got approval for different copy [here](https://mozilla.slack.com/archives/C4DCECLGJ/p1721340571446629?thread_ts=1721331665.902959&cid=C4DCECLGJ).

Reply:
I've updated the copy.